<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="pkg_name" value="apriltag"/>
    <arg name="node_name" default="apriltag_node"/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex: experiment_1."/>


    <group ns="$(arg veh)">
        <remap from="apriltag_node/cam" to="$(arg veh)/camera_node/image/compressed"/>
        <remap from="apriltag_node/cam_info" to="$(arg veh)/camera_node/camera_info"/>
        <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" required="true">
            <param name="veh" value="$(arg veh)" />
        </node>
    </group>
</launch>
